### DISCUSSION TITLE
kernel does not support WireGuard!

### DISCUSSION THREAD

**Discussion by darrepac** [wg-easy/wg-easy#267] [General]
Hello

I am using the official docker run command on OMV5 system (Debian 10) and I got the following error:
```2022-05-28T17:23:25.992Z Server Listening on http://0.0.0.0:51821
2022-05-28T17:23:25.995Z WireGuard Loading configuration...
2022-05-28T17:23:25.997Z WireGuard Configuration loaded.
2022-05-28T17:23:25.998Z WireGuard Config saving...
2022-05-28T17:23:26.000Z WireGuard Config saved.
$ wg-quick down wg0
$ wg-quick up wg0
Error: WireGuard exited with the error: Cannot find device "wg0"
This usually means that your host's kernel does not support WireGuard!
    at /app/lib/WireGuard.js:63:19
    at async /app/lib/WireGuard.js:61:9
```
I am a bit surprised cause the system is not that old and kernel too... (4.19.0-20-amd64)
Is there a workaround?

**Comment by archont94** [comment]
It is old, Wireguard was added to kernel in version 5.6. Check this [issue](https://github.com/WeeJeWel/wg-easy/issues/327) and [fix for it](https://github.com/WeeJeWel/wg-easy/pull/328) - if this will be merged, then you will be able to use wireguard-go implementation even on old kernels.

**Comment by pheiduck** [comment]
There is a new PR in WIP: https://github.com/wg-easy/wg-easy/pull/702

**Reply by pheiduck** [reply]
@darrepac and @archont94 for testing: `ghcr.io/wg-easy/wg-easy:development`

**Comment by m0x61h0x64i** [comment]
My vps kernel does not update please help, ubuntu 22
